filecache/file_cache.py

```diff
@@ -250,6 +250,10 @@ def __init__(self,
 
         atexit.register(self._maybe_delete_cache)
 
+    @classmethod
+    def registered_scheme_prefixes(self) -> tuple(str):
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider fixing tuple type annotation syntax</b></div>
<div id="fix">

The return type annotation `tuple(str)` appears incorrect. In Python type hints, it should be `tuple[str, ...]` for a tuple of strings.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -254 +254 @@
 -    def registered_scheme_prefixes(self) -> tuple(str):
 +    def registered_scheme_prefixes(self) -> tuple[str, ...]:
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/SETI/rms-filecache/pull/24#issuecomment-2495300024>#fc43b8</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged

Reply:
Good suggestion.